python/py-pyobjc-qtkit/Portfile

```diff
@@ -2,62 +2,23 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
+PortGroup           obsolete 1.0
 
 name                py-pyobjc-qtkit
-python.rootname     pyobjc-framework-QTKit
 version             5.3
 revision            0
-categories-append   devel
-platforms           darwin
-license             MIT
-maintainers         nomaintainer
-
-description         QTKit wrappers for PyObjC
-long_description    This port provides wrappers for the QTKit framework for \
-                    use with PyObjC. QTKit is an modern, object-oriented \
-                    framework for working with QuickTime media in Cocoa \
-                    applications, and is a replacement for the older \
-                    Carbon-based Quicktime framework.
-homepage            https://pyobjc.readthedocs.io/
-
-master_sites        pypi:p/${python.rootname}/
-distname            ${python.rootname}-${version}
-
-checksums           rmd160  f7b5d1216944d5e52e3227f4e804722ae0681990 \
-                    sha256  d831c3d444364dc8b419db59dc023a4ff826392f1d602d36103085182a1e34dd \
-                    size    119683
-
-python.versions     27 35 36 37 38
+epoch               1
 
-if {${name} ne ${subport}} {
-    depends_build-append \
-                        port:py${python.version}-setuptools
-    depends_lib-append  port:py${python.version}-pyobjc \
-                        port:py${python.version}-pyobjc-quartz
-
-    patchfiles      pyobjc_setup.py.patch
-
-    if {${configure.sdkroot} eq ""} {
-        set sdkroot "/"
-    } else {
-        set sdkroot ${configure.sdkroot}
-    }
+categories          python devel
+license             MIT
 
-    build.env-append    SDKROOT=${sdkroot}
-    destroot.env-append SDKROOT=${sdkroot}
+replaced_by         py-pyobjc
 
-    platform darwin 8 {
-        depends_build
-        depends_lib
-        pre-fetch {
-            ui_error "${subport} @${version} requires Mac OS X 10.5 or later"
-            return -code error "incompatible Mac OS X version"
-        }
-    }
+description         obsolete wrapper for PyObjC
 
-    post-destroot {
-        xinstall -m 0644 -W ${worksrcpath} \
-            LICENSE.txt README.txt \
-            ${destroot}${prefix}/share/doc/${subport}
+foreach v {27 35 36 37 38} {
```

Comment:
please remove the py35 subport; you cannot set it to be replaced_by py35-pyobjc as that port doesn't define the py35 subport